insight-engine/latest/admin/monitor.md

```diff
@@ -177,6 +177,37 @@ Use the following setting to specify an option core for the report. If it is abs
 
 You can also fix index issues, check the index cache and backup individual indexes by using JMX. The status of the index can be checked using the JMX client on the **JMX MBeans > Alfresco > solrIndexes > `<store>`** tabs. The default view is the Solr core summary. The operations run the same consistency checks that are available by URL.
 
+### Troubleshooting - unsearchable content
```

Comment:
I thought to add it as an extra heading because the formatting in the Troubleshooting topic is very different. In the Troubleshooting you only have one-liners, and this is a longer procedure. Also, it is specified at the beginning of that topic that there is information on how to repair a transaction that failed to index only. Maybe adding "transaction failed to index" to the "Troubleshooting" section name and changing the heading level of the added section to two would be better? So that it's not a subtopic of the original Troubleshooting.

Reply:
I like your suggestion 👍🏼